tests/meltano/core/test_yaml.py

```diff
@@ -139,3 +140,47 @@ def test_mixed_numeric_types() -> None:
     assert "789" in output
     assert "0.001" in output
     assert "999999999.999999999" in output
+
+
+def test_yaml_width_prevents_line_wrapping() -> None:
+    """Test that long lines are not wrapped due to sys.maxsize YAML width."""
+    from meltano.core.yaml import yaml as yaml_instance
```

Comment:
It's preferable to put all imports at the top of the file, unless we need, for example, to avoid circular imports or to make certain code paths faster in the CLI.

Reply:
Noted, thanks! `pre-commit.ci` auto-fixed this as well, as we no longer need this import now that we are using `yaml.dump()` in the test instead.